### PR TITLE
Add Initial Salary Survey Results

### PR DESCRIPTION
**Pull request by ericholscher**
We have additional images coming in,
which will be added here later today hopefully.

### CODE REVIEW

File: docs/surveys/salary-survey/2019.rst [hunk 1]
```diff
@@ -0,0 +1,735 @@
...
+   | - USA           | 344             | 54.4%           | $93,600         |
+   +-----------------+-----------------+-----------------+-----------------+
+   | - Canada        | 42              | 6.6%            | $60,273         |
+   +-----------------+-----------------+-----------------+-----------------+
+   | Europe          | 164             | 25.9%           | $48,700         |
+   +-----------------+-----------------+-----------------+-----------------+
+   | - EU Member     | 112             | 17.2%           | $46,104         |
+   | States          |                 |                 |                 |
+   +-----------------+-----------------+-----------------+-----------------+
+   | - UK \*         | 36              | 5.5%            | $60,269         |
```
Comment (plaindocs) [inline docs/surveys/salary-survey/2019.rst]: Might be nice to rejig these as proper footnotes:

```
diff --git a/docs/surveys/salary-survey/2019.rst b/docs/surveys/salary-survey/2019.rst
index 3cf3b711..340fbe98 100644
--- a/docs/surveys/salary-survey/2019.rst
+++ b/docs/surveys/salary-survey/2019.rst
@@ -319,7 +319,7 @@ Salary by Region/Country
    | - EU Member     | 112             | 17.2%           | $46,104         |
    | States          |                 |                 |                 |
    +-----------------+-----------------+-----------------+-----------------+
-   | - UK \*         | 36              | 5.5%            | $60,269         |
+   | - UK [#f1]_     | 36              | 5.5%            | $60,269         |
    +-----------------+-----------------+-----------------+-----------------+
    | - Germany       | 24              | 3.7%            | $59,430         |
    +-----------------+-----------------+-----------------+-----------------+
@@ -340,8 +340,11 @@ Salary by Region/Country
    | Given           |                 |                 |                 |
    +-----------------+-----------------+-----------------+-----------------+
 
-\* including Scotland - no respondents were reported from Wales or
-Northern Ireland
+
+.. rubric:: Footnotes
+
+.. [#f1] including Scotland - no respondents were reported from Wales or
+ Northern Ireland
```

Comment (ericholscher) [reply docs/surveys/salary-survey/2019.rst]: Yea, it's just an initial HTML<->RST import. We might move forward w/ publishing HTML, so I wanted to wait to polish it until we see how it looks.

Comment (plaindocs) [reply docs/surveys/salary-survey/2019.rst]: Yeah, that is why I didn't go ahead and fix it all.

File: docs/surveys/salary-survey/2019.rst [hunk 2]
```diff
@@ -0,0 +1,735 @@
...
+Out of the 594 valid responses:
+
+.. table::  Most Common Geographical Locations
+
+   +---------------+---------------------------+
+   | Location      | Percentage of Respondents |
+   +===============+===========================+
+   | United States | 58%                       |
+   +---------------+---------------------------+
+   | Canada        | 43                        |
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
   | Canada        | 7%                        |
```

File: docs/surveys/salary-survey/2019.rst [hunk 3]
```diff
@@ -0,0 +1,735 @@
...
+-------------
+
+Note: as 97% (632) of respondents reporting working between 30 and 60
+hours per week - a “full time” role - the 3% reporting fewer than 30
+hours have been omitted from the figures in this section.
+
+The median salary across all regions was $74,500 (meaning half of the
+respondents earned more, and half earned less).
+
+This figure is perhaps not very representative as it takes into account
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
This figure is not very representative as it does not take into account
```

File: docs/surveys/salary-survey/2019.rst [hunk 4]
```diff
@@ -0,0 +1,735 @@
...
+the socio-economic situation of the countries of the very highest
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
the socio-economic situation in the countries of the very highest
```

File: docs/surveys/salary-survey/2019.rst [hunk 5]
```diff
@@ -0,0 +1,735 @@
...
+   Figure: Education Level
+
+--------------
+
+.. h.hpkr7a16phmf:
+
+Location
+--------
+
+55 respondents left this question blank or provided a non-quantifiable
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
18 respondents left this question blank or provided a non-quantifiable
```

File: docs/surveys/salary-survey/2019.rst [hunk 6]
```diff
@@ -0,0 +1,735 @@
...
+.. figure:: images/15.png
+   :alt: Figure: Education Level
+
+   Figure: Education Level
+
+--------------
+
+.. h.hpkr7a16phmf:
+
+Location
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
Geographical Location
```

File: docs/surveys/salary-survey/2019.rst [hunk 7]
```diff
@@ -0,0 +1,737 @@
...
+
+.. h.dfv7pdw0gekn:
+
+Proportion of role officially related to documentation
+------------------------------------------------------
+
+65% of respondents reported that documentation was the whole of their
+official job description, and 30% reported that it was part of their
+official job description. This overwhelming majority (95% of all
+responses) is not surprising, given the focus of the Write The Docs
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
responses) is not surprising, given the focus of the Write the Docs
```

File: docs/surveys/salary-survey/2019.rst [hunk 8]
```diff
@@ -0,0 +1,737 @@
...
+community.
+
+What was interesting, however, was the statistically relevant number of
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
What was interesting was the statistically relevant number of
```

File: docs/surveys/salary-survey/2019.rst [hunk 9]
```diff
@@ -0,0 +1,737 @@
...
+respondents choosing the other two options: 4% said that although
+documentation was not part of their job description, they were expected
+to perform documentation-related tasks anyway, and 1.5% said that
+although documentation was not part of their job description, and they
+were not expected to perform documentation-related tasks, they did
+anyway.
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion

```

File: docs/surveys/salary-survey/2019.rst [hunk 10]
```diff
@@ -0,0 +1,737 @@
...
+official job description. This overwhelming majority (95% of all
+responses) is not surprising, given the focus of the Write The Docs
+community.
+
+What was interesting, however, was the statistically relevant number of
+respondents choosing the other two options: 4% said that although
+documentation was not part of their job description, they were expected
+to perform documentation-related tasks anyway, and 1.5% said that
+although documentation was not part of their job description, and they
+were not expected to perform documentation-related tasks, they did
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
were not expected to perform documentation-related tasks, they did.
```

File: docs/surveys/salary-survey/2019.rst [hunk 11]
```diff
@@ -0,0 +1,737 @@
...
+earners (out of the top 10 salary values, 9 were from the US and one
+from the UK) and the very lowest (the bottom 10 salary values were from
+ India, Asia, and Eastern Europe).
+
+Figures grouped into regions make a more useful baseline from which to
+determine what constitutes a “fair” salary.
+
+.. h.mqgdllu8gaap:
+
+Salary by Region/Country
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
Median Salary by Region and Country
```

File: docs/surveys/salary-survey/2019.rst [hunk 12]
```diff
@@ -0,0 +1,737 @@
...
+   +--------------+---------------+
+   | Texas        | $92,000       |
+   +--------------+---------------+
+   | Oregon       | $89,000       |
+   +--------------+---------------+
+   | Virginia     | $78,625       |
+   +--------------+---------------+
+
+.. figure:: images/map.svg
+   :alt: Figure: Median Salary by Region/Country
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
   :alt: Figure: Median Salary by Region and Country
```

File: docs/surveys/salary-survey/2019.rst [hunk 13]
```diff
@@ -0,0 +1,737 @@
...
+
+   Figure: Median Salary by Region/Country
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
   Figure: Median Salary by Region and Country
```

File: docs/surveys/salary-survey/2019.rst [hunk 14]
```diff
@@ -0,0 +1,737 @@
...
+
+--------------
+
+.. h.ldoc9nicw4s7:
+
+Additional Benefits
+-------------------
+
+Health insurance and vacation time were the most common benefits
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
Health insurance and paid vacation time were the most common benefits
```

File: docs/surveys/salary-survey/2019.rst [hunk 15]
```diff
@@ -0,0 +1,737 @@
...
+.. figure:: images/13.svg
+   :alt: Figure: Gender Identity
+
+   Figure: Gender Identity
+
+--------------
+
+.. h.6ascb2ek053u:
+
+Education Level
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
Highest Education Level Achieved
```

File: docs/surveys/salary-survey/2019.rst [hunk 16]
```diff
@@ -0,0 +1,737 @@
...
+---------------
+
+Note: one respondent skipped this question
+
+95% of respondents had completed a college or university degree or
+higher. Those attending technical college numbered less than 3%, and
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
higher. Those completing technical college numbered less than 3%, and
```

File: docs/surveys/salary-survey/2019.rst [hunk 17]
```diff
@@ -0,0 +1,737 @@
...
+those who completed high school only (including those who did some
+college but did not achieve a formal qualification) accounted for the
+remaining fraction.
+
+.. figure:: images/15.svg
+   :alt: Figure: Education Level
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
   :alt: Figure: Highest Education Level Achieved
```

File: docs/surveys/salary-survey/2019.rst [hunk 18]
```diff
@@ -0,0 +1,737 @@
...
+
+   Figure: Education Level
```
Comment (goatlady) [inline docs/surveys/salary-survey/2019.rst]:
```suggestion
   Figure: Highest Education Level Achieved
```

**Review by goatlady — COMMENTED** [review]
A few typo/wording fixes, a few headline and caption clarifications

File: docs/_static/css/survey.css [hunk 19]
```diff
@@ -0,0 +1,26 @@
+@media screen{
+  #write-the-docs-salary-survey-2019-results hr{margin-top:40px}
+  #write-the-docs-salary-survey-2019-results table caption{padding:0.5em}
+
+  #write-the-docs-salary-survey-2019-results table#id3 thead tr th:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id6 thead tr th:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 20]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id3 tbody tr td:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id6 tbody tr td:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 21]
```diff
@@ -0,0 +1,26 @@
+@media screen{
+  #write-the-docs-salary-survey-2019-results hr{margin-top:40px}
+  #write-the-docs-salary-survey-2019-results table caption{padding:0.5em}
+
```
Comment (goatlady) [inline docs/_static/css/survey.css]: #write-the-docs-salary-survey-2019-results div.figure {margin-top:1em}
#write-the-docs-salary-survey-2019-results div.figure p.caption{text-align:center;font-size:120%}

File: docs/_static/css/survey.css [hunk 22]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id3 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id3 tbody tr td:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id12 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id12 tbody tr td:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id15 thead tr th:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id19 thead tr th:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 23]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id15 tbody tr td:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id19 tbody tr td:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 24]
```diff
@@ -0,0 +1,26 @@
+@media screen{
+  #write-the-docs-salary-survey-2019-results hr{margin-top:40px}
+  #write-the-docs-salary-survey-2019-results table caption{padding:0.5em}
+
+  #write-the-docs-salary-survey-2019-results table#id3 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id3 tbody tr td:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id12 thead tr th:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id16 thead tr th:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 25]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id12 tbody tr td:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id16 tbody tr td:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 26]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id15 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id15 tbody tr td:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id16 thead tr th:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css:8]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id20 thead tr th:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 27]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id16 tbody tr td:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css:9]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id20 tbody tr td:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 28]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id17 thead tr th:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id21 thead tr th:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 29]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id17 tbody tr td:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id21 tbody tr td:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 30]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id23 thead tr th:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id27 thead tr th:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 31]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id23 tbody tr td:nth-child(2)
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id27 tbody tr td:nth-child(2)
```

File: docs/_static/css/survey.css [hunk 32]
```diff
@@ -0,0 +1,26 @@
...
+  {text-align:center}
+
+  #write-the-docs-salary-survey-2019-results table#id10 thead th{vertical-align:middle;white-space:nowrap}
+  #write-the-docs-salary-survey-2019-results table#id10 thead tr th:nth-child(2),
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id14 thead tr th:nth-child(2),
```

File: docs/_static/css/survey.css [hunk 33]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id10 tbody tr td:nth-child(2)
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id14 tbody tr td:nth-child(2)
```

File: docs/_static/css/survey.css [hunk 34]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id15 tbody tr td:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id16 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id16 tbody tr td:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id17 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id17 tbody tr td:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id23 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id23 tbody tr td:nth-child(2)
+  {text-align:center}
+
+  #write-the-docs-salary-survey-2019-results table#id10 thead th{vertical-align:middle;white-space:nowrap}
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id14 thead th{vertical-align:middle;white-space:nowrap}
```

File: docs/_static/css/survey.css [hunk 35]
```diff
@@ -0,0 +1,26 @@
...
+  #write-the-docs-salary-survey-2019-results table#id10 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id10 tbody tr td:nth-child(2)
+  {text-align:right}
+
+  #write-the-docs-salary-survey-2019-results table#id12 tbody tr td:nth-child(2){vertical-align:middle}
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
  #write-the-docs-salary-survey-2019-results table#id16 tbody tr td:nth-child(2){vertical-align:middle}
  
  /* salary by region table */
 #write-the-docs-salary-survey-2019-results table#id13 thead th{vertical-align:middle}

#write-the-docs-salary-survey-2019-results table#id13 thead th em{font-style:normal;font-size:90%;font-weight:normal;display:block}

#write-the-docs-salary-survey-2019-results table#id13 th,
#write-the-docs-salary-survey-2019-results table#id13 td
{vertical-align:middle} 

#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(3),
#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(4),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(3),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(4)
{text-align:center}

#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(5),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(5)
{text-align:right}
```

**Review by goatlady — COMMENTED** [review]
fixing changed table ids

File: docs/_static/css/survey.css (hunk 21 above)
Comment (plaindocs) [inline docs/_static/css/survey.css]:
```suggestion
#write-the-docs-salary-survey-2019-results div.figure {margin-top:1em}
#write-the-docs-salary-survey-2019-results div.figure p.caption{text-align:center;font-size:120%}
```

File: docs/_static/css/survey.css [hunk 36]
```diff
@@ -0,0 +1,27 @@
...
+  #write-the-docs-salary-survey-2019-results table#id27 tbody tr td:nth-child(2)
+  {text-align:center}
+
+  #write-the-docs-salary-survey-2019-results table#id14 thead th{vertical-align:middle;white-space:nowrap}
+  #write-the-docs-salary-survey-2019-results table#id14 thead tr th:nth-child(2),
+  #write-the-docs-salary-survey-2019-results table#id14 tbody tr td:nth-child(2)
+  {text-align:right}
+
+  #write-the-docs-salary-survey-2019-results table#id16 tbody tr td:nth-child(2){vertical-align:middle}
+
```
Comment (goatlady) [inline docs/_static/css/survey.css:26]: #write-the-docs-salary-survey-2019-results table#id13 thead th{vertical-align:middle}

#write-the-docs-salary-survey-2019-results table#id13 thead th em{font-style:normal;font-size:90%;font-weight:normal;display:block}

#write-the-docs-salary-survey-2019-results table#id13 th,
#write-the-docs-salary-survey-2019-results table#id13 td
{vertical-align:middle} 

#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(3),
#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(4),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(3),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(4)
{text-align:center}

#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(5),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(5)
{text-align:right}

Comment (plaindocs) [inline docs/_static/css/survey.css:26]:
```suggestion


#write-the-docs-salary-survey-2019-results table#id13 thead th{vertical-align:middle}

#write-the-docs-salary-survey-2019-results table#id13 thead th em{font-style:normal;font-size:90%;font-weight:normal;display:block}

#write-the-docs-salary-survey-2019-results table#id13 th,
#write-the-docs-salary-survey-2019-results table#id13 td
{vertical-align:middle}

#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(3),
#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(4),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(3),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(4)
{text-align:center}

#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(5),
#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(5)
{text-align:right}

```

File: docs/_static/css/survey.css [hunk 37]
```diff
@@ -0,0 +1,46 @@
...
+
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
#write-the-docs-salary-survey-2019-results table#id13 colgroup col:nth-child(1){width:21%}
#write-the-docs-salary-survey-2019-results table#id13 colgroup col:nth-child(2){width:24%}
#write-the-docs-salary-survey-2019-results table#id13 colgroup col:nth-child(3){width:25%}
#write-the-docs-salary-survey-2019-results table#id13 colgroup col:nth-child(4){width:15%}
#write-the-docs-salary-survey-2019-results table#id13 colgroup col:nth-child(5){width:15%}
```

File: docs/_static/css/survey.css [hunk 38]
```diff
@@ -0,0 +1,50 @@
...
+#write-the-docs-salary-survey-2019-results table#id13 colgroup col:nth-child(3){width:25%}
+#write-the-docs-salary-survey-2019-results table#id13 colgroup col:nth-child(4){width:15%}
+#write-the-docs-salary-survey-2019-results table#id13 colgroup col:nth-child(5){width:15%}
+#write-the-docs-salary-survey-2019-results table#id13 thead th{vertical-align:middle}
+
+#write-the-docs-salary-survey-2019-results table#id13 thead th em{font-style:normal;font-size:90%;font-weight:normal;display:block}
+
+#write-the-docs-salary-survey-2019-results table#id13 th,
+#write-the-docs-salary-survey-2019-results table#id13 td
+{vertical-align:middle}
```
Comment (goatlady) [inline docs/_static/css/survey.css]:
```suggestion
{vertical-align:middle;padding:0.25em 0.4em}
```

**Comment by ericholscher** [comment]
👍 on merge -- I can't 👍 it though because I started the PR :)

**Review by plaindocs — APPROVED** [review]
A-OK :shipit:

**Comment by plaindocs** [comment]
Or now holding off for a minor image fix?

File: docs/_static/css/survey.css [hunk 39]
```diff
@@ -0,0 +1,50 @@
...
+#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(3),
+#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(4),
+#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(3),
+#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(4)
+{text-align:center}
+
+#write-the-docs-salary-survey-2019-results table#id13 thead tr th:nth-child(5),
+#write-the-docs-salary-survey-2019-results table#id13 tbody tr td:nth-child(5)
+{text-align:right}
+
```
Comment (goatlady) [inline docs/_static/css/survey.css:49]: #write-the-docs-salary-survey-2019-results #geographical-location ul.simple li{float:left;width:33%}
#write-the-docs-salary-survey-2019-results #geographical-location ul.simple{overflow:auto}

Comment (plaindocs) [inline docs/_static/css/survey.css:49]:
```suggestion

#write-the-docs-salary-survey-2019-results #geographical-location ul.simple li{float:left;width:33%}
#write-the-docs-salary-survey-2019-results #geographical-location ul.simple{overflow:auto}
```

**Comment by plaindocs** [comment]
:boom:

**Pull request merged**